stack-citationapi/recipes/deploy-internalapi.rb

```diff
@@ -0,0 +1,45 @@
+node['deploy'].each do |application, deploy|
+
+  case application
+  when 'crossref_service'
+    next unless allow_deploy(application, 'crossref_service', 'crossref-www')
+  when 'highbeam'
+    next unless allow_deploy(application, 'highbeam')
+  when 'jstor'
+    next unless allow_deploy(application, 'jstor')
+  when 'oclc_eswitch'
+    next unless allow_deploy(application, 'oclc_eswitch', 'oclc-eswitch')
+  when 'proquest'
+    next unless allow_deploy(application, 'proquest')
+  when 'recommend'
+    next unless allow_deploy(application, 'recommend')
+  when 'worldcat'
+    next unless allow_deploy(application, 'worldcat')
+  when 'yahooboss'
+    next unless allow_deploy(application, 'yahooboss')
+  when 'citation_apis'
```

Comment:
All of the above except `'citation_apis'` can be removed - I cleaned this up last week :)